mne/io/brainvision/brainvision.py

```diff
@@ -286,9 +294,12 @@ def _read_annotations_brainvision(fname, sfreq='auto'):
     orig_time = _str_to_meas_date(date_str)
 
     if sfreq == 'auto':
-        vhdr_fname = op.splitext(fname)[0] + '.vhdr'
-        logger.info("Finding 'sfreq' from header file: %s" % vhdr_fname)
-        _, _, _, info = _aux_vhdr_info(vhdr_fname)
+        hdr_fname = op.splitext(fname)[0] + '.vhdr'
+        # if vhdr file does not exist assume that the format is ahdr
+        if not op.exists(hdr_fname):
+            hdr_fname = op.splitext(fname)[0] + '.ahdr'
```

Comment:
Why not use `ahdr_format` here? I think it would be cleaner.

Reply:
The problem is that ahdr file doesn't always mean amrk file for markers, since you can have ahdr for header and vhdr for marker… 
Probably the best thing would be saving the files during the initialization when the header file is read, without making any assumption on the extension of marker file.

Reply:
I looked again at the code and I think it is not possible, because `_read_annotations_brainvision` is not called directly by the BrainVision object, but it is called by the more general `read_annotations`, therefore it has only the marker file and the frequency as inputs.
Unfortunately, even if it is not really clean, I think checking if the file exists is the most conservative option.
In the rest of the code the information on the format of the marker file is not needed.

Reply:
OK!